src/templates/views/partials/featured-items.hbs

```diff
@@ -1,7 +1,7 @@
 {{#each featuredItems}}
 <ul class="featured-items">
 	<li class="featured-item card">
-		<a href="{{ aboutUs.url }}" class="featured-item__link cover">
+		<a href='page/who-we-are#racial_equity_statement' class="featured-item__link cover">
```

Comment:
- The hard-coded link works in this case, but if the underlying `Feauted Item` model is updated in Keystone or the URL changes this link will break.  We can combine the existing approach with the new section link to make this less brittle: `href="{{ aboutUs.url }}#racial_equity_statement"`
- Make sure to use double quotes instead of single to maintain consistency in .hbs templates.